src/cpp/ext/filters/census/grpc_plugin.cc

```diff
@@ -75,93 +75,103 @@ ::opencensus::trace::Span GetSpanFromServerContext(
     grpc::ServerContext* context) {
   if (context == nullptr) return opencensus::trace::Span::BlankSpan();
 
-  return reinterpret_cast<const grpc::CensusContext*>(context->census_context())
+  return reinterpret_cast<const grpc::experimental::CensusContext*>(
+             context->census_context())
       ->Span();
 }
 
 // These measure definitions should be kept in sync across opencensus
 // implementations--see
 // https://github.com/census-instrumentation/opencensus-java/blob/master/contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcMeasureConstants.java.
-::opencensus::tags::TagKey ClientMethodTagKey() {
+::opencensus::tags::TagKey experimental::ClientMethodTagKey() {
```

Comment:
Instead of adding an `experimental::` prefix to each of these functions, let's just wrap them in a `namespace experimental { ... }` block.